src/Item.py

```diff
@@ -0,0 +1,39 @@
+
+images = {
+    "sword": "assets/sword.png",
+    "spear": "assets/spear.png",
+}
```

Comment:
Can you add these default stats and images? Along with anything else in the `assets` folder
```python
        weapons = [
            {"name": "sword", "image": "./assets/sword.jpg", "KEY": [20, 30, 10]},
            {"name": "spear", "image": "./assets/spear.jpg", "KEY": [30, 10, 20]},
        ]
        armor = [
            {"name": "shield", "image": "./assets/shield.jpg", "KEY": [30, 10, 20]},
            {"name": "gloves", "image": "./assets/gloves.jpg", "KEY": [10, 10, 10]},
        ]```